app/src/main/res/layout/activity_edit_question.xml

```diff
@@ -45,40 +50,68 @@
                 android:layout_height="wrap_content"
                 android:layout_gravity="center"
                 android:padding="@dimen/answer_text_helper_padding"
-                android:text="@string/add_answer_helper_text"
+                android:text="@string/choose_answer_format"
                 android:textSize="@dimen/helper_text_size" />
 
-            <TableLayout
-                android:id="@+id/answers_table"
+            <LinearLayout
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:layout_gravity="center"> <!-- stretch the second column of the layout-->
-
-                <TableRow
-
-                    android:id="@+id/row0"
-                    android:layout_width="fill_parent"
-                    android:layout_height="wrap_content">
-
-                    <ImageButton
-                        android:id="@+id/add_button"
-                        android:layout_width="wrap_content"
-                        android:layout_height="wrap_content"
-                        android:background="@color/colorButton"
-                        android:padding="@dimen/answer_button_padding"
-                        app:srcCompat="@drawable/ic_add_black" />
-                </TableRow>
-
-            </TableLayout>
-            <Button
-                android:id="@+id/button_done_editing"
+                android:layout_gravity="center"
+                android:orientation="horizontal">
+
+                <ImageButton
```

Comment:
We should maybe add some visual feedback when the user has defined an answer format. Right now there is no information that tells you that you have successfully entered an answer format.

Reply:
Thanks, i just did that!